.github/ISSUE_TEMPLATE.md

```diff
@@ -19,3 +19,11 @@ Describe in more detail the problem you have been experiencing, if necessary.
 ## Link to Appium logs
 
 Create a [gist](https://gist.github.com) which is a paste of your _full_ Appium logs, and link them here. Do _not_ paste your full Appium logs here, as it will make this issue very long and hard to read! If you are reporting a bug, _always_ include Appium logs!
+
+
+## Code To Reproduce Issue
+
+Please git clone https://github.com/appium/sample-code and use one of your favourite languages and sample apps to reproduce issue.
+In case similar scenario is missing in sample-code , please submit PR with one of the sample apps provided.
```

Comment:
* I would say `In case a similar scenario...` (add 'a')
* And likewise, `please submit a PR`